imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -1,133 +1,182 @@
 default_attrs: &default
   # Assumed values for all variable attrs unless overwritten
   DISPLAY_TYPE: time_series
-  DEPEND_0: epoch
-  FILLVAL: -9223372036854775808
-  FORMAT: I12
-  VALIDMIN: 0
-  VALIDMAX: 9223372036854775807
-  VAR_TYPE: data
   UNITS: ' '
 
-de_default_attrs: &de_default
-  DISPLAY_TYPE: time_series
-  FILLVAL: -9223372036854775808
-  FORMAT: I12
+default_uint8_attrs: &default_uint8
+  <<: *default
+  FILLVAL: 255
+  FORMAT: I3
   VALIDMIN: 0
-  VALIDMAX: 9223372036854775807
+  VALIDMAX: 255
+  dtype: uint8
+
+default_uint16_attrs: &default_uint16
+  <<: *default
+  FILLVAL: 65535
+  FORMAT: I5
+  VALIDMIN: 0
+  VALIDMAX: 65535
+  dtype: uint16
+
+default_uint32_attrs: &default_uint32
+  <<: *default
+  FILLVAL: 4294967295
+  FORMAT: I10
+  VALIDMIN: 0
+  VALIDMAX: 4294967295
+  dtype: uint32
+
+default_uint64_attrs: &default_uint64
```

Comment:
I found out the hard way that CDF doesn't have a CDF_UINT8 datatype, so I think that we need to use `int64` only, not `uint64`.